teuthology/task/install.py

```diff
@@ -65,7 +65,10 @@
     'python-ceph',
     'rbd-fuse',
 ]
-
+RHVERSION={}
+RHVERSION['ceph']={
+    '1.3.0':1    
+}
```

Comment:
@zmc 
 I will remove this for now and will add when we actually use it, I think its better to see this in use during upgrade tests.